packages/framework-provider-aws-infrastructure/src/index.ts

```diff
@@ -1,6 +1,12 @@
+import { ProviderInfrastructure, PluginDescriptor } from '@boostercloud/framework-types'
+import { loadPlugin } from './infraestructure-plugin'
 import { deploy, nuke } from './infrastructure'
+export { InfrastructurePlugin } from './infraestructure-plugin'
```

Comment:
Typo here, its `infrastructure`

```suggestion
export { InfrastructurePlugin } from './infrastructure-plugin'
```